core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -955,4 +955,10 @@ <h2 class="activity-title e2e-test-exp-summary-tile-title">
      width: fit-content;
     }
   }
+    .mat-card>:first-child, .mat-card-content>:first-child {
```

Comment:
Hey @ramyaram2092 , can you please remove the extra space before `.mat-card>`. Thanks! Once you have done that, feel free to assign me back to the PR by commenting `@SanjaySajuJacob PTAL`